WordPress/Sniffs/WP/UseCapabilitiesNotRolesSniff.php

```diff
@@ -0,0 +1,234 @@
+		'edit_private_posts'          => true,
+		'read_private_posts'          => true,
+		'delete_private_pages'        => true,
+		'edit_private_pages'          => true,
+		'read_private_pages'          => true,
+		'edit_published_posts'        => true,
+		'upload_files'                => true,
+		'publish_posts'               => true,
+		'delete_published_posts'      => true,
+		'edit_posts'                  => true,
+		'delete_posts'                => true,
+		'read'                        => true,
+		'level_10'                    => true,
+		'level_9'                     => true,
+		'level_8'                     => true,
+		'level_7'                     => true,
+		'level_6'                     => true,
+		'level_5'                     => true,
+		'level_4'                     => true,
+		'level_3'                     => true,
+		'level_2'                     => true,
+		'level_1'                     => true,
+		'level_0'                     => true,
+		'create_sites'                => true,
+		'delete_sites'                => true,
+		'manage_network'              => true,
+		'manage_sites'                => true,
+		'manage_network_users'        => true,
+		'manage_network_plugins'      => true,
+		'manage_network_themes'       => true,
+		'manage_network_options'      => true,
+		'delete_site'                 => true,
+		'upgrade_network'             => true,
+		'setup_network'               => true,
+		'upload_plugins'              => true,
+		'upload_themes'               => true,
+		'customize'                   => true,
+		'add_users'                   => true,
+		'install_languages'           => true,
+		'update_languages'            => true,
+		'deactivate_plugins'          => true,
+		'upgrade_php'                 => true,
+		'export_others_personal_data' => true,
+		'erase_others_personal_data'  => true,
+		'manage_privacy_options'      => true,
+		'edit_categories'             => true,
+		'delete_categories'           => true,
+		'manage_post_tags'            => true,
+		'edit_post_tags'              => true,
+		'delete_post_tags'            => true,
+		'edit_css'                    => true,
+		'assign_categories'           => true,
+		'assign_post_tags'            => true,
+	);
+
+	/**
+	 * Process the parameters of a matched function.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param int    $stackPtr        The position of the current token in the stack.
+	 * @param array  $group_name      The name of the group which was matched.
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array  $parameters      Array with information about the parameters.
+	 *
+	 * @return void
+	 */
+	public function process_parameters( $stackPtr, $group_name, $matched_content, $parameters ) {
+
+		$position = $this->target_functions[ $matched_content ];
+		if ( ! isset( $parameters[ $position ] ) ) {
+			return;
+		}
+
+		$matched_parameter = $this->strip_quotes( $parameters[ $position ]['raw'] );
```

Comment:
This will only work when the `$capability` is passed as a single text string.

Note: If the warning is to be removed/disabled by default (see my other remark), this should be fine to leave as is.